terraform/modules/self-service/self_service_app.tf

```diff
@@ -20,3 +20,9 @@ resource "aws_iam_user_policy_attachment" "self_service_user_policy_attachment"
   user       = "${aws_iam_user.self_service_user.name}"
   policy_arn = "${aws_iam_policy.self_service_user_write_to_bucket.arn}"
 }
+
+resource "aws_vpc_endpoint" "self_service_s3" {
```

Comment:
there is already an S3 vpc endpoint in `terraform/modules/hub/vpc.tf` - I'm not sure we need another one?

Reply:
In particular, if we deploy the self-service app to the `internal` subnets, we'll get to reuse the S3 endpoint for free.

Reply:
The endpoint in hub has permissions to read as well as write. I guess I'm a little unsure whether limiting the connecting user to have read permissions only is enough?

The Self Service app won't be deployed to the Hub for a little while yet and in the mean time we would like to begin integrating the publishing of metadata from the app and hub config reading it.